managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -30,14 +31,30 @@ const mapDispatchToProps = (dispatch) => {
     },
 
     fetchKMSConfigList: () => {
-      return dispatch(fetchAuthConfigList()).then((response) =>
+      return dispatch(fetchAuthConfigList()).then((response) => 
         dispatch(fetchAuthConfigListResponse(response.payload))
-      );
+      )
+      .catch((err) => {
+        dispatch(addToast({
+          toast: {
+            type: 'error',
+            description: 'Error occured while fetching config.',
+            position: "bottom-right",
```

Comment:
let's not pass the CSS properties as part of props. We can use some other key to define the CSS. Like in this case we can use the title to define the CSS property for the toast.

Reply:
Done